src/common/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+ov_deprecated_no_errors()
```

Comment:
it would be great to remove deprecated API from common code. I supposed current folder does not hav dependency on extensions, Data, InputInfo..
Or at least we need to make suppressions more specific.